### PR TITLE
minor changes to and solution for **Reverting A Commit** challenge

### PR DESCRIPTION
**Pull request by kekoziar**
I tightened up the language, added some spaces for readability, and added a solution to the challenge.

### CODE REVIEW

**Comment by sarah-peter** [comment]
This would solve issue #603 about the missing solution.

File: _episodes/05-history.md [hunk 1]
```diff
@@ -409,6 +412,16 @@ moving backward and forward in time becomes much easier.
 > 4. Type in the new commit message.
 >
 > 5. Save and close
+> 
+> 
+> > ## Solution
+> > 
+> > The command `git log` lists project history with commit IDs.  
+> > 
+> > The command `git show HEAD~1` shows changes made at the commit one before `HEAD` and lists
```
Comment (munkm) [inline _episodes/05-history.md]: Maybe I'm misinterpreting the exercise prompt, but wouldn't her last commit to the project repository be shown with `git show HEAD`, not `git show HEAD~1`?

File: _episodes/05-history.md [hunk 2]
```diff
@@ -390,16 +390,19 @@ moving backward and forward in time becomes much easier.
...
-> commit that reverses Jennifer's erroneous commit. Therefore `git revert` is
-> different to `git checkout [commit ID]` because `git checkout` returns the
-> files within the local repository to a previous state, whereas `git revert`
-> reverses changes committed to the local and project repositories.  
+> Jennifer is collaborating with colleagues on her Python script.  She
+> realizes her last commit to the project's repository contained an error, and 
+> wants to undo it.  Jennifer wants to undo correctly so everyone in the project's
+> repository gets the correct change. The command `git revert [erroneous commit ID]` will create a 
+> new commit that reverses the erroneous commit.  
+> &nbsp;
```
Comment (munkm) [inline _episodes/05-history.md]: Is there a reason these spacing commands are used at lines 398 and 403? We don't use this formatting elsewhere in the lesson.

Comment (kekoziar) [reply _episodes/05-history.md:398]: It was added to improve readability, and make separate paragraphs more distinct.  I'll double-check the readability.

Comment (kekoziar) [reply _episodes/05-history.md]: @munkm, I removed the nbsp.

File: _episodes/05-history.md (hunk 1 above)
Comment (kekoziar) [reply _episodes/05-history.md]: Yes, I think when I wrote this I was thinking `checkout` instead of `revert`.  I fixed the error.

**Comment by chendaniely** [comment]
Hi all. @kekoziar asked me to take a look at this during this month's co-working session.
I read through it and it all looks like she addressed all the comments that were made in the past, so it all looks good to be merged :)

I didn't actually render the content on my local machine. but the text all *seems* to be correctly formatted.

Hopefully this helps!

**Review by chendaniely — APPROVED** [review]
I didn't render the lesson, but this all looks good!

Thanks for filling out the solution 👍

**Comment by tobyhodges** [comment]
Confirming that the page builds fine and the formatting is good.

Screenshot 2021-08-05 at 11 02 51

**Comment by kekoziar** [comment]
Thanks @chendaniely  and @tobyhodges  for the assistance!

**Pull request merged**